src/torcontrol.cpp

```diff
@@ -404,7 +404,6 @@ void TorController::get_socks_cb(TorControlConnection& _conn, const TorControlRe
     Assume(resolved.IsValid());
     LogDebug(BCLog::TOR, "Configuring onion proxy for %s\n", resolved.ToStringAddrPort());
 
-    // With m_randomize_credentials = true, generates unique SOCKS credentials per proxy connection (e.g., Tor).
     // Prevents connection correlation and enhances privacy by forcing different Tor circuits.
```

Comment:
This looks weird out of context. If we're going to update this yet again, maybe something like:
```c++
// Add Tor as proxy for .onion addresses.
// Enable stream isolation to prevent connection correlation and enhance privacy, by forcing a different Tor circuit for every connection.
// For this to work, the IsolateSOCKSAuth flag must be enabled on SOCKSPort (which is the default, see the IsolateSOCKSAuth section of Tor's manual page).
```
The ugly, and probably temporary URL has been bothering me.